### ISSUE TITLE
Exception throw when calling get on ManagementSystem

### ISSUE THREAD

**Issue by crakjie** [thinkaurelius/titan#961]
```
mgmt = titanGraph.getManagementSystem()
mgmt.get("storage.backend")
```

Throw

```
play.api.UnexpectedException: Unexpected exception[IllegalArgumentException: Can only accept global options: root.storage.hostname]
    at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:166) ~[play_2.11-2.3.7.jar:2.3.7]
    at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130) ~[play_2.11-2.3.7.jar:2.3.7]
    at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
    at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130) ~[play_2.11-2.3.7.jar:2.3.7]
    at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128) ~[play_2.11-2.3.7.jar:2.3.7]
Caused by: java.lang.IllegalArgumentException: Can only accept global options: root.storage.hostname
    at com.google.common.base.Preconditions.checkArgument(Preconditions.java:148) ~[guava-16.0.1.jar:na]
    at com.thinkaurelius.titan.diskstorage.configuration.BasicConfiguration.verifyOption(BasicConfiguration.java:41) ~[titan-core-0.5.2.jar:na]
    at com.thinkaurelius.titan.diskstorage.configuration.BasicConfiguration.has(BasicConfiguration.java:48) ~[titan-core-0.5.2.jar:na]
    at com.thinkaurelius.titan.diskstorage.configuration.UserModifiableConfiguration.get(UserModifiableConfiguration.java:59) ~[titan-core-0.5.2.jar:na]
    at com.thinkaurelius.titan.graphdb.database.management.ManagementSystem.get(ManagementSystem.java:1111) ~[titan-core-0.5.2.jar:na]
```

I was expecting the value I have set in the configuration, is it normal?

**Comment by vijayrudraraju** [comment]
I'm getting the same thing, haven't figured it out yet...

**Comment by dalaro** [comment]
@crakjie @vijayrudraraju the ManagementSystem interface currently only talks to the storage backend.  It does not consider the local configuration file.  Config options marked `LOCAL` in the [manual's config option reference](http://s3.thinkaurelius.com/docs/titan/current/titan-config-ref.html) live only in the local configuration file and will effectively be invisible to ManagementSystem.  `storage.backend` is `LOCAL`.  That's why this IAE is thrown.

It would be reasonably straightforward to tweak ManagementSystem to allow reading `LOCAL` config options, but writing would be trickier.  Titan currently treats the config file as read-only.  There might be a case for doing both read & write functionality for `LOCAL` options, but I'm concerned that implementing read but not write would just lead to more surprises in the config interface (and option mutability levels cause quite a bit of that already).  That +read but -write asymmetry is kind of justifiable for `FIXED` options just because of the name of that mutability level, but with `LOCAL`, it would definitely be surprising to have an exception thrown when attempting to `set` a new value.  Even if we implemented r/w for `FIXED` options, they'd still be kind of odd, since they're the only options that are scoped to a particular copy of a config file on a filesystem instead of the shared storage backend.

In summary:
- I don't think this is a bug
- We could consider adding R/W for `LOCAL` options, but that's probably better handled in a different issue (or even better, a PR)
